lib/withSentry.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Raven from "raven-js";
+
+const SENTRY_DSN = "https://b23c49f878524c8eaf98fb47340d2448@sentry.io/1082251";
```

Comment:
This isnt some sort of key, right?

Reply:
It is the public portion of the key, ideally this would be injected through ENV variables but it would still be in the bundle served to the client. All it does is prevent it from being embedded in other domains and it is up to us in sentry to configure the correct domains to track from. 
<img width="1043" alt="screen shot 2018-04-05 at 7 41 46 am" src="https://user-images.githubusercontent.com/867334/38364001-d1ba22c6-38a4-11e8-88f9-482852b540b1.png">